support-frontend/assets/pages/contributions-landing/components/ContributionAmountChoices.jsx

```diff
@@ -41,6 +41,31 @@ const choiceCardGrid = css`
   }
 `;
 
+const choiceCardGridProductSetAbTestStyles = css`
+  ${until.mobileLandscape} {
+    > div {
+      width: 100%;
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+      grid-gap: 8px;
+
+      > label {
+        margin-bottom: 0px;
+      }
+    }
+  }
+`;
+
+const choiceCardGridProductSetAbTestFullWithOtherStyles = css`
+${choiceCardGridProductSetAbTestStyles}
+
+> div > label:last-of-type {
+  grid-column-start: 1;
+  grid-column-end: 3;
+}
```

Comment:
Ideally we'd be able to put these styles in the `cssOverrides` of the `ChoiceCard` but that doesn't work, so it has to be on the parent choice card so we have to get hacky again.

It doesn't work because the `ChoiceCard` jsx looks a little like

```ts
<>
  <input />
  <label />
</>
```
and the overrides get put on the `input`, but we need them on the `label`